src/library/entities/Library.java

```diff
@@ -17,65 +17,65 @@
-	private Date lOaN_DaTe;
-	
-	private Map<Integer, Book> CaTaLoG;
-	private Map<Integer, Member> MeMbErS;
-	private Map<Integer, Loan> LoAnS;
-	private Map<Integer, Loan> CuRrEnT_LoAnS;
-	private Map<Integer, Book> DaMaGeD_BoOkS;
+	private static final String LIBRARY_FILE = "library.obj"; //Changed lIbRaRyFiLe to LIBRARY_FILE
+	private static final int LOAN_LIMIT = 2; //Changed lOaNlImIt to LOAN_LIMIT
+	private static final int LOAN_PERIOD = 2; //Changed loanPeriod to LOAN_PERIOD
+	private static final double FINE_PER_DAY = 1.0; //Changed FiNe_PeR_DaY to FINE_PER_DAY
+	private static final double MAX_FINES_OWED = 1.0; //Changed maxFinesOwed to MAX_FINES_OWED
+	private static final double DAMAGE_FEE = 2.0; //Changed damageFee to DAMAGE_FEE
+	
+	private static Library self; //Changed SeLf to self
+	private int bookId; //Changed bOoK_Id to bookId
+	private int memberId; //Changed mEmBeR_Id to memberId
+	private int loanId; //Changed lOaN_Id to loanId
+	private Date loanDate; //Changed lOaN_DaTe to loanDate
+	
+	private Map<Integer, Book> catalog; //Changed CaTaLoG to catalog
+	private Map<Integer, Member> members; //Changed MeMbErS to members
+	private Map<Integer, Loan> loans; //Changed LoAnS to loans
+	private Map<Integer, Loan> currentLoans; //Changed CuRrEnT_LoAnS to currentLoans
+	private Map<Integer, Book> damagedBooks; //Changed DaMaGeD_BoOkS to damagedBooks
 	
 
 	private Library() {
-		CaTaLoG = new HashMap<>();
-		MeMbErS = new HashMap<>();
-		LoAnS = new HashMap<>();
-		CuRrEnT_LoAnS = new HashMap<>();
-		DaMaGeD_BoOkS = new HashMap<>();
-		bOoK_Id = 1;
-		mEmBeR_Id = 1;		
-		lOaN_Id = 1;		
-	}
-
-	
-	public static synchronized Library GeTiNsTaNcE() {		
-		if (SeLf == null) {
-			Path PATH = Paths.get(lIbRaRyFiLe);			
-			if (Files.exists(PATH)) {	
-				try (ObjectInputStream LiBrArY_FiLe = new ObjectInputStream(new FileInputStream(lIbRaRyFiLe));) {
+		catalog = new HashMap<>();
+		members = new HashMap<>();
+		loans = new HashMap<>();
+		currentLoans = new HashMap<>();
+		damagedBooks = new HashMap<>();
+		bookId = 1;
+		memberId = 1;		
+		loanId = 1;		
+	}
+
+	
+	public static synchronized Library getInstance() {	//Changed GeTiNsTaNcE to getInstance 	
+		if (self == null) {
+			Path path = Paths.get(LIBRARY_FILE);	//Changed Path PATH to PATH path		
+			if (Files.exists(path)) {	
+				try (ObjectInputStream libraryFile = new ObjectInputStream(new FileInputStream(LIBRARY_FILE));) { //Changed LiBrArY_FiLe to libraryFile
 			    
-					SeLf = (Library) LiBrArY_FiLe.readObject();
-					Calendar.gEtInStAnCe().SeT_DaTe(SeLf.lOaN_DaTe);
-					LiBrArY_FiLe.close();
+					self = (Library) libraryFile.readObject();
+					Calendar.getInstance().setDate(self.loanDate); //Calendar.gEtInStAnCe().SeT_DaTe changed to getInstance().setDate to match method in Calendar
+					libraryFile.close();
 				}
 				catch (Exception e) {
 					throw new RuntimeException(e);
 				}
 			}
-			else SeLf = new Library();
+			else self = new Library();

```

Comment:
Technically curly brackets can be used here so probably should be.

Reply:
Added curly brackets in latest commit.